xcode/Subconscious/Shared/Services/AddressBookService.swift

```diff
@@ -185,62 +185,62 @@ actor AddressBook<Sphere: SphereProtocol> {
         return name
     }
     
-    func getPetname(petname: Petname) async throws -> Did? {
+    func getPetname(petname: Petname.Name) async throws -> Did? {
         return try? await self.sphere.getPetname(petname: petname)
     }
 
     nonisolated func getPetnamePublisher(
-        petname: Petname
+        petname: Petname.Name
     ) -> AnyPublisher<Did?, Error> {
         Future.detached(priority: .utility) {
             try await self.getPetname(petname: petname)
         }
         .eraseToAnyPublisher()
     }
 
-    func setPetname(did: Did, petname: Petname) async throws {
+    func setPetname(did: Did, petname: Petname.Name) async throws {
         try await sphere.setPetname(did: did, petname: petname)
     }
 
     nonisolated func setPetnamePublisher(
         did: Did,
-        petname: Petname
+        petname: Petname.Name
     ) -> AnyPublisher<Void, Error> {
         Future.detached {
           try await self.setPetname(did: did, petname: petname)
         }
         .eraseToAnyPublisher()
     }
 
-    func unsetPetname(petname: Petname) async throws {
+    func unsetPetname(petname: Petname.Name) async throws {
         try await sphere.setPetname(did: nil, petname: petname)
     }
 
     nonisolated func unsetPetnamePublisher(
-        petname: Petname
+        petname: Petname.Name
     ) -> AnyPublisher<Void, Error> {
         Future.detached(priority: .utility) {
           try await self.unsetPetname(petname: petname)
         }
         .eraseToAnyPublisher()
     }
 
-    func listPetnames() async throws -> [Petname] {
+    func listPetnames() async throws -> [Petname.Name] {
```

Comment:
I think this depends on the semantics of what `Petname` as a type actually represents. 

One major question to determine this is whether it makes sense to say that a user's _nickname_ is a `Petname` or whether it's just a `Name`. If it is, then a `Petname` is really just a certain string format. 

BUT it seems like in Noosphere we often refer to a `Petname` as a _resolvable_ name, not just a well-formatted string. I actually don't feel strongly either way, but because of the inertia I would maybe lean towards the Noosphere model?

Worth saying: we could still return a first-order `Petname` (as opposed to a `Name`) from some or all of these Noosphere methods. Perhaps the Noosphere layer only accepts `Petname`s but the app sometimes has to model other, more general `Name`s?